syntax = "proto3";

package auth;

service AuthService {
    rpc GenerateToken(UserCredentials) returns (TokenResponse);
}

message UserCredentials {
    string username = 1;
    string password = 2;
}

message TokenResponse {
    string token = 1;
}